pystiche_papers/ulyanov_et_al_2016/_modules.py

```diff
@@ -150,6 +217,25 @@ def conv_block(
 
 
 class JoinBlock(nn.Module):
+    r"""JoinBlock from :cite:`ULVL2016`.
```

Comment:
I think this is a better fit here.

```suggestion
    r"""JoinBlock from :cite:`ULVL2016` without upsampling.
```